### PR TITLE
Add new person data, introduce nick_name_original

### PR DESCRIPTION
**Pull request by scr0llbaer9000**
Now on to new game data that I submit in parallel non-conflicting PRs.

This one has new person data required for my new programme entries.

I have also added all person entries I found from https://github.com/Marathony and from https://github.com/StukovTTV as I work with that dataset already when I create new programme entries locally.

I found we also need to introduce and cater for `nick_name_original`.

### CODE REVIEW

File: res/database/Default/database_people.xml [hunk 1]
```diff
@@ -3271,7 +3756,55 @@
 		<person id="SpeedMinister-person-id040" first_name="Douggy" last_name="Eves" first_name_original="Douglas" last_name_original="Adams" gender="1" birthday="1952-03-11" deathday="2001-05-11" imdb_id="nm0010930" creator="8605" created_by="speedminister" />
 		<person id="SpeedMinister-person-id041" first_name="Joe" last_name="Nedlloyd" first_name_original="John" last_name_original="Lloyd" gender="1" birthday="1951-09-30" deathday="" imdb_id="nm0516032" creator="8605" created_by="speedminister" />
 
-		<!-- from scr0llbaer - TODO - but here are already the people turned into references in the descriptions of already existing programmes -->
+		<!-- from Stukov -->
+
+		<person id="Stukov_GeorgeMiller_" first_name="Gabriel" last_name="Millson" first_name_original="George" last_name_original="Miller" imdb_id="nm0004306" gender="1" job="1" topgenre="2" birthday="1945-03-03" deathday="" />
```
Comment (nittka) [inline res/database/Default/database_people.xml:3761]: please remove topgenre

File: res/database/Default/database_people.xml [hunk 2]
```diff
@@ -3271,7 +3756,55 @@
...
+		<person id="Stukov_HonorDavisPye" first_name="Holton" last_name="Daves-Pie" first_name_original="Honor" last_name_original="Davis-Pye" imdb_id="nm10675999" gender="2" job="96" birthday="2009-12-11" deathday="" />
+		<person id="Stukov_ReyMysterio" first_name="Ray" last_name="Mistico" first_name_original="Rey" last_name_original="Mysterio" imdb_id="nm0347595" gender="1" job="4192"  birthday="1974-12-11" deathday=""/>
+		<person id="Stukov_ThomasCarter" first_name="Travis" last_name="Cartridge" first_name_original="Thomas" last_name_original="Carter" imdb_id="nm0141961" gender="1" job="69"  birthday="1953-07-17" deathday=""/>
+		<person id="Stukov_RandalKleiser" first_name="Randy" last_name="Kliesing" first_name_original="Randal" last_name_original="Kleiser" imdb_id="nm0459170" gender="1" job="1"  birthday="1946-07-20" deathday=""/>
+		<person id="Stukov_JoeDante" first_name="John" last_name="Danti" first_name_original="Joe" last_name_original="Dante" imdb_id="nm0001102" gender="1" job="1" topgenre="12"  birthday="1946-11-28" deathday=""/>
```
Comment (nittka) [inline res/database/Default/database_people.xml:3768]: remove topgenre

File: res/database/Default/database_people.xml [hunk 3]
```diff
@@ -3271,7 +3756,55 @@
...
+		<person id="Stukov_BradfordDillman" first_name="Brantley" last_name="Dellmen" first_name_original="Bradford" last_name_original="Dillman" imdb_id="nm0226947" gender="1" job="34" topgenre="12" birthday="1930-04-14" deathday="2018-01-16" />
```
Comment (nittka) [inline res/database/Default/database_people.xml:3769]: remove topgenre

**Review by nittka — CHANGES_REQUESTED** [review]

File: res/database/Default/database_people.xml (hunk 3 above)
Comment (scr0llbaer9000) [reply res/database/Default/database_people.xml]: I suppose you mean only in in the `insignificantpeople` entries. Done.

**Review by nittka — APPROVED** [review]
I have no formal complaints; @GWRon ?

In the long run, removing unused, non-bookable persons on game start sound good but will be non-trivial, because it is possible they are not used as cast but in title/description/news (potentially even within variables.

A local approach would be logging non-bookable Persons not used in cast.

**Comment by GWRon** [comment]
What I dislike a bit - and this might affect other DB files - is the "ID" in the form "Stukov_Name". I am more keen of stuff like "person-firstNameLastName" or similar. Especially if it is placed in the "official" database (and even more if it is added the first time). Author contribution is already there via "created_by" and "created". If you want to ensure "uniqueness" then simply have it "person-5digits-firstNameLastName" - clashes with persons having the same name should be very very rare then (in the US acting guild each name can only exist once - but this does not apply to other countries).

For movie titles I guess IDs are less "important" in these regards as you do not reuse them - so most probably only the author itself will have to do with the chosen (GU)ID.

I kinda want to avoid _others_ to have to write "Stukov_" (or "ron_") all the time (poking and peeking "this is an entry of XYZ!"). If it was hidden in a GUI I would not say a thing - but with our current way of editing the XML files it is a bit ...hmm.

Anyways. It is just a personal opinion - and it seems to only affect "person"-data. And ... while writing it I am now no longer sure  if I should even write this text or not. But I did and maybe you have something to say about it too.

**Comment by nittka** [comment]
I guess this is (at least partially) my fault. The documentation for IDs suggests this format (taken from the original Wiki entries concerning the database) regardless of the entry type

https://github.com/TVTower/Documentation/blob/master/database_de/main.md#id

As long as an entry is located in a user file, this ID-format does not pose a problem. The question is what should happen when entries become "official". And as you said, persons (and maybe roles) are critical because unlike other entries they are very likely referenced and changing the ID is not nice (breaking existing entries potentially all over the place).

option 1: no change, IDs are just IDs if there happens to be a "meaningful" prefix, ignore it (UUIDs are used as well and they are even less readable)
option 2: refactor person and role IDs not to have a user prefix at all (person-xxx, role-yyy) even for user files, so that migrating persons to the official database can be done without having to update the references.

As there is a cleanup going on anyway, I would vote for option 2. However, this may mean that other databases out there may break...

**Comment by nittka** [comment]
@GWRon : What is your opinion (or rather decision). I would not mind a cleanup, but I am not opposed to not changing the person IDs. I still think, the current state could be merged.

**Comment by GWRon** [comment]
There is no need to change the IDs now. This is something which could be done somewhen in the future.

**Pull request merged**